theme/static/js/hs-vue/left-header-app.js

```diff
@@ -133,6 +141,15 @@ Vue.component('add-author-modal', {
             if (!userId) {
                 vue.addAuthorError = "Select a user to add as an author";
                 return;
+            }else {
+                const alreadyExists = leftHeaderApp.$data.authors.map(function (author) {
```

Comment:
This works, but it's inefficient since the array is traversed twice. We could do something  like this instead:
`
const alreadyExists = leftHeaderApp.$data.authors.some(function (author) {
                        return author.profileUrl === "/user/" + userId + "/";
                    })
`
This is irrelevant in terms of performance here given the small datasets, but makes the code cleaner, easier to maintain and is good practice for the future.

Reply:
This makes sense, thanks!